config/default.yml

```diff
@@ -573,6 +573,12 @@ RSpec/MessageSpies:
   VersionAdded: '1.9'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/MessageSpies
 
+RSpec/MinitestAssertions:
+  Description: Check if using Minitest matchers.
+  Enabled: pending
+  VersionAdded: '2.16'
```

Comment:
We're merging swiftly lately, but just in case - `<<next>>`?

Reply:
That sounds better. We updated to `<<next>>`.